xml/Makefile.am

```diff
@@ -210,20 +210,18 @@ pacemaker-%.rng: $(CIB_build_copies) rng-helper Makefile.am
 
 # Dynamically generated CIB schema listing all pacemaker versions
 #
-# @COMPAT none is deprecated since 2.1.8, as is validate-with being optional
+# @COMPAT "none" is deprecated since 2.1.8
```

Comment:
Currently, the XSL transform to 4.0 adds `validate-with="pacemaker-4.0"` if `validate-with` is unset. I think it would be straightforward to change that to `validate-with="none"` if desired, to minimize behavioral changes.

That may not be worthwhile, since we plan to drop "none" when we can.

Reply:
Right, don't want to shift someone to none only to drop it later

Reply:
This change actually breaks the transformation mentioned above, since unset `validate-with` now fails to validate. It's probably better to just live with that.